scaler/queue_pinger.go

```diff
@@ -45,7 +46,7 @@ func newQueuePinger(
 		for {
 			select {
 			case <-pingTicker.C:
-				if err := pinger.requestCounts(); err != nil {
+				if err := pinger.requestCounts(context.TODO()); err != nil {
```

Comment:
can you use at least a `context.Background()` here. it would be better to pass a `context.Context` to this function though, so if that's easy enough, it's preferable

Reply:
Done, passing context to pinger, but creating as background in main